product_docs/docs/epas/15/planning/deployment_options/aws_epas.mdx

```diff
@@ -23,16 +23,16 @@ To deploy an EDB Postgres Advanced Server instance on AWS:
 
 1. On the **Launch an instance** page, select **Choose an Amazon Machine Image(AMI)**.
 
-1. On the **Choose an Amazon Machine Image(AMI)** page, go to **AWS Marketplace AMIs** tab, type **EDB** in the search bar and choose the EDB Postgres Advanced Server image. 
+1. On the **Choose an Amazon Machine Image(AMI)** page, go to the **AWS Marketplace AMIs** tab. Enter **EDB** in the search bar.  
```

Comment:
@nidhibhammar It looked like the deleted text is repeated in the next step, which is why I deleted it. Please verify.